app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :update_post_like_counter
+
```

Comment:
- [ ] Instead of calling `after_create` callback on your `update_post_like_counter` I would suggest you to use `after_save` callback because `after_create` callback is invoked only once, immediately after the record is first created and does not get triggered when updating an existing record. On the other hand `after_save` callback is called every time you save the object, regardless of whether it’s a new or existing record.